Installation/cmake/modules/CGAL_CreateSingleSourceCGALProgram.cmake

```diff
@@ -18,6 +18,18 @@ function(create_single_source_cgal_program firstfile )
 
     add_executable(${exe_name} ${all})
 
+    if(BUILD_TESTING)
+      if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cin")
+        set(ARGS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cin")
+      elseif(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cmd")
+        file(STRINGS "${CMAKE_CURRENT_SOURCE_DIR}/${exe_name}.cmd" ARGS LIMIT_COUNT 1)
```

Comment:
I don't think this will be sufficient. In some cmd, you have several lines and each line is actually a run